src/cascadia/TerminalApp/MinMaxCloseControl.xaml

```diff
@@ -10,12 +10,283 @@
     HorizontalAlignment="Left"
     VerticalAlignment="Top"
     Orientation="Horizontal"
-    d:DesignHeight="300"
+    d:DesignHeight="36"
     d:DesignWidth="400">
 
-    <Grid x:Name="Content" ></Grid>
-    <Border Height="36.0" MinWidth="175.0" x:Name="DragBar" Background="{ThemeResource ApplicationPageBackgroundThemeBrush}" DoubleTapped="DragBar_DoubleTapped"/>
-    <Button Height="36.0" Width="40.0" x:Name="Minimize" Background="{ThemeResource ApplicationPageBackgroundThemeBrush}" Click="Minimize_Click">_</Button>
-    <Button Height="36.0" Width="40.0" x:Name="Maximize" Background="{ThemeResource ApplicationPageBackgroundThemeBrush}" Click="Maximize_Click">-</Button>
-    <Button Height="36.0" Width="40.0" x:Name="Close" Background="{ThemeResource ApplicationPageBackgroundThemeBrush}" Click="Close_Click">X</Button>
+    <!-- NOTE: All paths are size 11x11 (or 11x1) because their content, when centered inside them,
+         is less blurry when aligned to the half pixel. -->
+    
+    <StackPanel.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.ThemeDictionaries>

```

Comment:
This is missing high contrast colors. Strongly recommend adding a HighContrast dictionary as well and making these resource references to [theme brushes](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/xaml-theme-resources#the-xaml-color-ramp-and-theme-dependent-brushes) .